palette/src/encoding/p3.rs

```diff
@@ -0,0 +1,277 @@
+//! The P3 color space(s) and standards.
+
+use core::marker::PhantomData;
+
+use crate::{
+    encoding::{FromLinear, IntoLinear, Srgb},
+    luma::LumaStandard,
+    num::{Powf, Real},
+    rgb::{Primaries, RgbSpace, RgbStandard},
+    white_point::{Any, WhitePoint, D65},
+    Mat3, Xyz, Yxy,
+};
+
+/// The white point of DCI-P3 (Theatrical) is based on a projector with a xenon bulb
+/// with a color temperature of ~6300K
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct XenonBulb;
+
+impl<T: Real> WhitePoint<T> for XenonBulb {
+    fn get_xyz() -> Xyz<Any, T> {
+        Xyz::new(
+            T::from_f64(0.314 / 0.351),
+            T::from_f64(1.0),
+            T::from_f64(0.335 / 0.351),
+        )
+    }
+}
```

Comment:
It may be better to bake this into the `DciP3` type, unless it's a generally established/standardized white point. Or give it a DCI or P3 related name. I'm generally careful with suggesting that something is, in this case, the canonical approximation of a xenon bulb.